data_parser.py

```diff
@@ -0,0 +1,84 @@
+import pandas as pd
+import numpy as np
```

Comment:
numpy is not in requirements

Reply:
Numpy is a part of tensorflow package already